Assignment2.yaml

```diff
@@ -0,0 +1,415 @@
+swagger: "2.0"
+info:
+  description: "Starwars API."
+  version: ""
+  title: "SWAPI"
+host: "swapi.co"
+basePath: "/api"
+tags:
+- name: "Vehicle"
+  description: "Everything about your Vehicle"
+  externalDocs:
+    description: "Find out more"
+    url: "http://swapi.co/documentation#vehicles"
+- name: "People"
+  description: "Everything about your People"
+  externalDocs:
+    description: "Find out more"
+    url: "http://swapi.co/documentation#people"
+- name: "Starship"
+  description: "Everything about your Starship"
+  externalDocs:
+    description: "Find out more"
+    url: "http://swapi.co/documentation#starships"
+- name: "Species"
+  description: "Everything about your Species"
+  externalDocs:
+    description: "Find out more"
+    url: "http://swapi.co/documentation#species"
+schemes:
+- "http"
+paths:
+  /Vehicle:
+    get:
+      tags:
+      - "Vehicle"
+      summary: "Find list of vehicles"
+      description: "Returns a List of vehicle"
+      produces:
+      - "application/json"
+      parameters:
+      - name: "name"
+        in: "query"
+        required: false
+        type: "string"
+      - name: "model"
+        in: "query"
+        required: false
+        type: "string"
+      responses:
+        200:
+          description: "successful operation"
+          schema:
+            type: "array"
+            items:
+             $ref: "#/definitions/Vehicle"
+  /Vehicle/{VehicleId}:
+    get:
+      tags:
+      - "Vehicle"
+      summary: "Find Vehicle by ID"
+      description: "Returns a single vehicle"
+      produces:
+      - "application/json"
+      parameters:
+      - name: "VehicleId"
+        in: "path"
+        description: "ID of Vehicle to return"
+        required: true
+        type: "number"
+      responses:
+        200:
+          description: "successful operation"
+          schema:
+            $ref: "#/definitions/Vehicle"
+  /Vehicle/schema:
+    get:
+      tags:
+      - "Vehicle"
+      summary: "Find Vehicle Schema"
+      description: "Returns a json schema"
+      produces:
+      - "application/json"
+      responses:
+        200:
+          description: "successful operation"
+          schema:
+            $ref: "#/definitions/Vehicle"
```

Comment:
Incorrect Response Schema. Check the Response by running the URL in your postman.